packages/scheduler-sequelize/test/recurrence_generators_test.ts

```diff
@@ -1,8 +1,8 @@
-import moment, { Moment } from 'moment-timezone';
+import moment from 'moment-timezone';
 import { expect } from 'chai';
```

Comment:
this already tests if the library works for lunartick rules and the rrule library, it ensures that it returns correct recurrences

also copied over to recurrence_generators_test in scheduler-prisma